homeassistant/components/knx/climate.py

```diff
@@ -184,20 +184,20 @@ def async_add_entities_config(hass, config, async_add_entities):
         min_temp=config.get(CONF_MIN_TEMP),
         max_temp=config.get(CONF_MAX_TEMP),
         mode=climate_mode,
+        on_off_invert=config.get(CONF_ON_OFF_INVERT),
```

Comment:
Please use `dict[key]` for required config keys and keys with default config schema values.

Reply:
FullCheck Pylint failed with error: homeassistant/components/knx/climate.py:168:14: E1123: Unexpected keyword argument 'on_off_invert' in constructor call (unexpected-keyword-arg).
This is because a new version of the XKNX library has not been released yet. How to solve it?

Reply:
We need a bump of the xknx library and update the version that we use in home assistant.

So first we wait for the xknx release.

Reply:
New XKNX has been released. How can I restart the check with new version of XKNX (last check failed with error: FullCheck Pylint failed with error: homeassistant/components/knx/climate.py:168:14: E1123: Unexpected keyword argument 'on_off_invert' in constructor call (unexpected-keyword-arg).

Reply:
Please rebase on latest dev branch.